.eslintrc.js

```diff
@@ -106,7 +106,6 @@ module.exports = {
     'no-use-before-define': 'off',
     'valid-typeof': 'off',
     'no-undef': 'off',
-    'no-control-regex': 'off',
```

Comment:
Is there a point to this rule if the only file affected is ignored?

Reply:
Ensures we don't do this in our source code.